examples/basic/package.json

```diff
@@ -12,19 +12,26 @@
     "@babel/runtime": "^7.14.0",
```

Comment:
This file changes does not need to be merged and should be removed before approval